benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -118,9 +133,9 @@ def get_sub_dict(dictionary: dict, key: str, value: str):
             else:
                 file_components = os.path.splitext(filename)
                 template_file = f'{file_components[0]}_template{file_components[1]}'
-            with open(os.path.join(workload_dir_path, 'internal_data', template_file)) as f:
+            with open(os.path.join(workload_dir_path, template_file)) as f:
                 template = Template(f.read())
-            answer[filename] = template.render(render_data)
+            answer[filename] = template.render(render_data) + "\n"
```

Comment:
Why we need a newline at the end of render ?

Reply:
Linux/UNIX convention is for all lines, including the last line, to be newline-terminated.

Reply:
from what I've seen, the templates files are generated without EOL at EOF,
it's best practise to have it https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline